tests/network/vmi_networking.go

```diff
@@ -399,13 +426,10 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		It("[test_id:1775]should not request a tun device", func() {
 			By("Creating random VirtualMachineInstance")
 			autoAttach := false
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-			// Remove the masquerade interface to use the default bridge one
-			vmi.Spec.Domain.Devices.Interfaces = nil
-			vmi.Spec.Networks = nil
+			vmi := libvmi.NewAlpine()
 			vmi.Spec.Domain.Devices.AutoattachPodInterface = &autoAttach
```

Comment:
Not worth it to make this a `libvmi` option?

Reply:
Grepping for it, I see only 3 instances where it is used in tests. I would not bother, it does not seem to have "potential".
But I would also not reject such a change if someone proposes it.

Reply:
I'm OK with leaving it as is, I was just trying to figure out what's the criteria for making something an option.